error-reporting/api/ErrorReporting.csproj

```diff
@@ -1,101 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why did you remove this file? It means that you won't be able to open the project on full Visual Studio...

Reply:
Or you moved the file?